wolfssl/wolfcrypt/settings.h

```diff
@@ -178,7 +191,7 @@
 #include <wolfssl/wolfcrypt/visibility.h>
 
 #ifdef WOLFSSL_USER_SETTINGS
-    #include "user_settings.h"
+    #include "../user_settings_folder/user_settings.h"
```

Comment:
This change is not okay. This will break many people's code. The proper fix is to add this to your own include path in your WICED project. `CFLAGS="-I../user_settings_folder"`.